### ISSUE TITLE
Issue with multiple include path in gitconfig

### ISSUE THREAD

**Issue by guyzmo** [gitpython-developers/GitPython#626]
cf guyzmo/git-repo#153 when it edits the `gitconfig` where there is multiple includes, it does not save the modifications in one of the includes.

**Comment by Byron** [comment]
@guyzmo Would you be able and willing to contribute the fix?

**Comment by guyzmo** [comment]
ah, on that one I'm not sure how to solve this… how to correctly change a value and place it back in the included file it comes from?

We might need some brainstorm over here to figure a fix out

**Comment by Byron** [comment]
Yeah, that looks like a major overhaul on how the config file system is working. Not even git can do it, after all one must specify the destination of any configuration file changes.
A valid fix would be to not allow changing files with includes - just silently discarding changes is definitely not the correct way of handling this.

**Comment by guyzmo** [comment]
Well, another way would be to just pull the section where you do your change back in the main config, or maybe have it twice (can a section be coumpounded?).
